### DISCUSSION TITLE
Get same private key for one email for any login methods

### DISCUSSION THREAD

**Discussion by yash-2523** [Web3Auth/Web3Auth#927] [Help]
I want to have users get one private key for one email, irrespective of their login method. Someone suggested me to have custom verifier and have email as a verifier field. But I am not sure how to do it. Can anyone explain this in detail. Thanks in advance!

**Comment by shahbaz17** [comment]
Hey @yash-2523 

This could be achieved by aggregating the verifiers.
Please take a look at https://web3auth.io/docs/custom-authentication/verifiers#aggregate-verifiers
